src/DataManagement/MonthlyReport/MonthlyReportResult.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ePiggyWeb.DataManagement.Goals;
+using ePiggyWeb.Utilities;
+
+namespace ePiggyWeb.DataManagement.MonthlyReport
+{
+    public class MonthlyReportResult
+    {
+        public DateTime Start { get; set; }
+        public DateTime End { get; set; }
+        public decimal Expenses { get; set; }
+        public decimal Income { get; set; }
+        public decimal Balance { get; set; }
+        public Importance BiggestCategory { get; set; }
+        public decimal BiggestCategorySum { get; set; }
+        public decimal NecessarySum { get; set; }
+        public decimal HowMuchBigger { get; set; }
+        public List<IGoal> SavedUpGoals { get; set; }
```

Comment:
Maybe this could be IGoalList?

Reply:
Now I can't remeber why, it was IGoalList at the start, but it didn't work. And after I changed it to List it started working :D